RELEASE.md

```diff
@@ -1,3 +1,75 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Cloud TPU:
+  * Experimental tf.data integration for Keras on Cloud TPUs.
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
```

Comment:
Can we add the following items to the Release notes? Thanks!
* DistributionStrategy
    * Add multi-GPU DistributionStrategy support in tf.keras. Users can now use `fit`, `evaluate` and `predict` to 
      distribute their model on multiple GPUs.
    * Add multi-worker  DistributionStrategy and standalone client support in Estimator. See  [README] 
      (https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute) for more details.

Reply:
Done, thanks!